features/bootstrap/DatasetViewContext.php

```diff
@@ -332,26 +329,26 @@ public function iShouldSeeTabWithTable($arg1, TableNode $table)
     }
 
     /**
-     * @Then I sould not see :arg1 tab with table
+     * @Then I should not see :arg1 tab with table
      */
-    public function iSouldNotSeeTabWithTable($arg1, TableNode $table)
+    public function iShouldNotSeeTabWithTable($arg1, TableNode $table)
     {
-        //| File name                                        | Sample ID  | Data Type         | File Format | Size      | Release date | link |
+        //| File Name                                        | Sample ID  | Data Type         | File Format | Size      | Release date | link |
         foreach($table as $row) {
             if ("Files" == $arg1) {
                 PHPUnit_Framework_Assert::assertFalse(
-                    $this->minkContext->getSession()->getPage()->hasContent($row['File name']), "File name match"
+                    $this->minkContext->getSession()->getPage()->hasContent($row['File Name']), "File Name match"
                 );
             }
             elseif("Sample" == $arg1) {
                 if ($row['Sample ID']) {
                     PHPUnit_Framework_Assert::assertFalse(
-                        $this->minkContext->getSession()->getPage()->hasContent($row['Sample ID']), "File name match"
+                        $this->minkContext->getSession()->getPage()->hasContent($row['Sample ID']), "File Name match"
```

Comment:
The `"File Name match"` String message seems wrong here when we compare this and above. The String message should probably be `"Sample ID match"`.